.github/workflows/scripts/magento-release.php

```diff
@@ -7,11 +7,15 @@
 class MagentoReleaseClient
 {
     const API_PATH = 'https://developer-api.magento.com/rest/v1';
+    const MAX_RETRIES = 5;
+    const RETRY_AFTER = 20;
```

Comment:
I would use max retries 3 and retry after 30

Reply:
OK, I've updated the values